dev_guide/managing_images.adoc

```diff
@@ -0,0 +1,538 @@
+when new images are added and react by performing a build or deployment,
+respectively.
+
+There are many ways image streams can be configured, depending on where the
+images' registries are located, how you want them to behave, and any security
+requirements. The following sections cover a range of these topics.
+
+[[understanding-image-tags]]
+== Understanding Image Tags
+
+Before jumping into image streams and their tags, it will help to first
+understand image tags in the context of Docker generally.
+
+Docker images can have names added to them that make it more intuitive what they
+contain. When interacting directly with images using the `docker` CLI, the
+`docker tag` command essentially adds an alias to an image that can consist of
+several parts. Those parts can include:
+
+----
+<registry_host>/<user_name>/<image_name>:<tag>
+----
+
+The `<user_name>` part could also refer to a project or namespace if the image
+is being stored in an {product-title} or Kubernetes environment.
+
+{product-title} provides the `oc tag` command, which is similar to the `docker
+tag` command, but operates on image streams instead of directly on images.
+
+[NOTE]
+====
+See Red Hat Enterprise Linux 7's
+https://access.redhat.com/documentation/en/red-hat-enterprise-linux-atomic-host/version-7/getting-started-with-containers/#creating_docker_images[Getting
+Started with Containers] documentation for more about tagging images directly
+using the `docker` CLI.
+====
+
+[[adding-tag]]
+== Adding Tags to Image Streams
+
+Keeping in mind that an image stream in {product-title} comprises one or more
+Docker images identified by tags, an image stream can also be configured so that
+a tag "tracks" another one.
+
+For example, to configure the *ruby* image's *latest* tag to always refer to the
+current image for the tag *2.0*:
+
+====
+----
+$ oc tag ruby:latest ruby:2.0
+----
+====
+
+[[tag-removal]]
+== Removing Tags from Image Streams
+
+You can also stop tracking a tag by removing the tag. For example, to stop
+tracking an existing *latest* tag:
+
+====
+----
+$ oc tag -d ruby:latest
+----
+====
+
+However, while the above command removes the tag from the image stream
+definition, it does not remove it from the image stream status. The image stream
+definition is user-defined, whereas the image stream status reflects the
+information the system has from the specification.
+
+To remove a tag completely from an image stream:
+
+====
+----
+$ oc delete istag/ruby:latest
+----
+====
+
+[[referencing-images-in-image-streams]]
+== Referencing Images in Image Streams
+
+When defining tags in an image stream, you can decide if the tag should be
+imported or just referenced from an image stream. This is achieved by setting
+`*reference*`. By default `*reference*` is *false*, which means the tag's
+metadata will be imported during the import. Setting it to *true* on the other
+hand, will mark the tag as a reference.
```

Comment:
@ncdc Having trouble figuring out where this `reference` could be set, and why/when. Is this the `reference` mentioned in http://file.rdu.redhat.com/~adellape/041216/edit_1501_imgmoves/rest_api/openshift_v1.html#v1-tagreference? Is there an example someplace of this in action?